travis/scripts/04-tests.sh

```diff
@@ -28,7 +28,7 @@ if [ -f "mvnw" ]; then
         -Dlogging.level.io.github.jhipster.sample=ERROR \
         -Dlogging.level.io.github.jhipster.travis=ERROR
 elif [ -f "gradlew" ]; then
-    ./gradlew test \
+    ./gradlew check \
```

Comment:
needs update to docs as well, also this is a breaking change, which is not nice for a non major release

Reply:
Both your points are valid. Doing a 4.9.0 would be the way to go I guess. Same as updating the maven docker plugin when it was changed from `docker:build` to `dockerfile:build`

Reply:
hmmm

Reply:
I'm also reluctant here: this will annoy a lot of people, and "test" is also the task name we use for Maven and Yarn

Reply:
I tend to agree with @jdubois 
I know its not the correct convention but I guess many of us are so used to running only the test tasks and since the test task still exists it will result it in we still running it and forgetting about the integration tests until something breaks in a CI

Reply:
Okay then. Let's keep test execute all unit and spring integration tests. What about cucumber? I would execute it via check and not test.

Reply:
that sounds fine